server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -55,14 +55,16 @@ export async function createGitService(
     const details = await tenantService.getTenant(tenantId, token, allowDisabledTenant);
     const customData: ITenantCustomDataExternal = details.customData;
     const writeToExternalStorage = !!customData.externalStorageData;
+    const storageName = customData.storageName;
```

Comment:
Don't we need a null check here?

Reply:
Do you mean the null check for storage name? :)
The storage name can be either a string or undefined, which will be handled at the resGitService.ts's undefined check logic.

Reply:
Could `customData` be `undefined` and cause some error like `TypeError: Cannot read properties of undefined `? We could just do
```
const storageName = customData?.storageName;
```
to avoid any issues. But Line 57 also does not do a null check, so I'm not sure if it's not required or if it's something we should also do in there (i.e. add null checks both in line 57 as well as 58)

Reply:
Thanks for the suggestions. I just add the null check for line 58 to be safe.